src/containerapp/azext_containerapp/_cloud_build_utils.py

```diff
@@ -61,7 +58,7 @@ def spin():
                     loop_counter = (loop_counter + 1) % 17
                     loading_bar_left_spaces_count = loop_counter - 9 if loop_counter > 9 else 0
                     loading_bar_right_spaces_count = 6 - loop_counter if loop_counter < 7 else 0
-                    spinner = f"[{' ' * loading_bar_left_spaces_count}{'=' * (7 - loading_bar_left_spaces_count - loading_bar_right_spaces_count)}{' ' * loading_bar_right_spaces_count}]"
+                    spinner = f"|{' ' * loading_bar_left_spaces_count}{'=' * (7 - loading_bar_left_spaces_count - loading_bar_right_spaces_count)}{' ' * loading_bar_right_spaces_count}|"
```

Comment:
What is the difference between `|` and `[`?

Reply:
This is purely a cosmetic issue. This change was made to copy Azure Dev new style that moved from showing spinners like `[===  ]` and instead is using `|===  |` now